sampler_test.go

```diff
@@ -27,6 +27,8 @@ var (
 	}
 
 	testDefaultSamplingProbability = 0.5
+
+	testMaxID = uint64(1) << 62
```

Comment:
split vars and consts. Most of these should be consts, but I don't think you can do that with maps, so those are the only vars.